packages/sozluk-content/next.config.mjs

```diff
@@ -0,0 +1,9 @@
+import { withContentlayer } from "next-contentlayer";
```

Comment:
I don't think this should be here, this package should not relay on something it's not using.

Reply:
I added this one since the build was failing when building `gql` app since it depends on some files generated with contentlayer under `sozluk-content` package.

this of course didn't fix the issue and the actual solution was adding `-r` flag to `pnpm run build` command so that it starts building `sozluk-content` before building `gql`. will test removing this today.

Reply:
just removed this `next-contentlayer` package thing, and the build process still works properly with `-r` flag added to `pnpm run build` command, until [this error.](https://github.com/mcanueste/monorepo/actions/runs/5648863844/job/15301975207)

this error is fixed by adding `"noImplicitAny": false` to `tsconfig.json` file, but don't know if this is wanted by the maintainers @Ketcap .